### DISCUSSION TITLE
Close editor after interactive rebase

### DISCUSSION THREAD

**Discussion by wilsonsilva** [zed-industries/zed#13796] [Feature Requests]
I use Zed for my interactive git rebases. When I save, I want the editor to close automatically when I save/close the current file. Is it possible? This is my git config:
```
[core]
	editor = zed -n -w
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
